### PR TITLE
Run linting #40

### PR DESCRIPTION
**Pull request by annawoodard**
Hi @yadudoc. Sorry for the gigantic PR, but I wanted to get this and #42 taken care of so that libsubmit will be multi-developer ready like Parsl. Commit 015b46d is all whitespace issues (that I fixed automatically with YAPF), and hopefully should not be very controversial. Non-whitespace/minor issues I split out into their own commits for you to review-- they all look like they were typos to me, but please have a look to be sure I wasn't missing something obvious.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,8 +7,10 @@ python:
 # command to install dependencies
 install:
     - pip install -r requirements.txt
+    - pip install flake8
```
Comment (yadudoc) [inline .travis.yml]: +1

File: .travis.yml [hunk 2]
```diff
@@ -7,8 +7,10 @@ python:
...
     - python setup.py install
 
 # command to run tests
 script:
-    - pip install -r test-requirements.txt
\ No newline at end of file
+    - pip install -r test-requirements.txt
+    - flake8 libsubmit/
```
Comment (yadudoc) [inline .travis.yml]: This is nice!

Comment (yadudoc) [reply .travis.yml]: Is flake8 in the test-requirements ?

Comment (annawoodard) [reply .travis.yml]: good catch @yadudoc, I've added it

**Pull request merged**